config/src/main/java/org/springframework/security/config/annotation/web/configurers/RememberMeConfigurer.java

```diff
@@ -267,6 +267,19 @@ public void init(H http) throws Exception {
 		validateInput();
 		String key = getKey();
 		RememberMeServices rememberMeServices = getRememberMeServices(http, key);
+		if (rememberMeServices instanceof AbstractRememberMeServices) {
+			 /*
```

Comment:
Please remove the comments. The code should be clear enough that comments are not needed.

Reply:
Removed